validation-test/stdlib/SetAlgebra.swift

```diff
@@ -0,0 +1,181 @@
+// RUN: %target-run-simple-swift
+// REQUIRES: executable_test
+
+import StdlibUnittest
+import StdlibCollectionUnittest
+
+extension MinimalEquatableValue : Hashable {
+  public var hashValue: Int {
+    return value
+  }
+}
+
+public struct DefaultedSetAlgebra<Element : Hashable> : SetAlgebra {
+  var store = Set<Element>()
+  public init(){}
+  public init(_ elements: [Element]) {
+    for el in elements {
+      store.insert(el)
+    }
+  }
+
+  public func contains(_ member: Element) -> Bool {
+    return store.contains(member)
+  }
+
+  public func union(_ other: DefaultedSetAlgebra) -> DefaultedSetAlgebra {
+    var defaultedSetAlgebra = DefaultedSetAlgebra()
+    defaultedSetAlgebra.store = store.union(other.store)
+    return defaultedSetAlgebra
+  }
+
+  public func intersection(_ other: DefaultedSetAlgebra) ->
+  DefaultedSetAlgebra {
```

Comment:
Please break the function signature before `->`, and add two extra spaces of indentation on continuation lines.

(Here and in other places below.)